hack/install.sh

```diff
@@ -86,21 +86,21 @@ install_containerd() {
   arch=$(uname -m)
   case $arch in
   x86_64 | amd64)
-    curl -sLSf https://github.com/containerd/containerd/releases/download/v1.5.4/containerd-1.5.4-linux-amd64.tar.gz | $SUDO tar -xvz --strip-components=1 -C /usr/local/bin/
+    curl -sLSf https://github.com/containerd/containerd/releases/download/v1.5.7/containerd-1.5.7-linux-amd64.tar.gz | $SUDO tar -xvz --strip-components=1 -C /usr/local/bin/
```

Comment:
@jsiebens do you think it makes sense to extract a variable `CONTAINERD_VER` here to make maintenance easier?